src/main/java/uk/gov/ch/repository/update/OverseasEntityBeneficialOwnerRepository.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.ch.repository.update;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import uk.gov.ch.model.update.OverseasEntityBeneficialOwner;
+
+import java.util.List;
+
+public interface OverseasEntityBeneficialOwnerRepository extends PagingAndSortingRepository<OverseasEntityBeneficialOwner, Long> {
+    String OE_INDIVIDUAL_BO_APPOINTMENT_ID = "5010";
+    String OE_OLE_BO_APPOINTMENT_ID = "5011";
+    String OE_GPA_BO_APPOINTMENT_ID = "5012";
+
+    @Query(value = "select "
```

Comment:
Just want to check that this SQL statement has been approved by a DBA (in case of performance issues when run against the LIVE DB)?

Reply:
Yes Mike Williams has approved the query, saying: 'looks ok to me...outer joins tend to slow queries down but one outer join in a query should be ok'